docs/team.rst

```diff
@@ -43,278 +34,275 @@ How teams operate and interact
 - **All membership and leadership roles are staffed by enthusiastic opt-in.** We want to prioritize the well-being of our contributors, and we're happy to have you if you're happy to contribute to the community. 
 - Each team operates independently except for actions that impact other teams or the community.
 - Each team is responsible for implementing and monitoring community guidelines, including CoC, within their team's area. 
-- All teams leads are included in the community council, which can propose and vote on internal organizational changes and processes. 
-- Team leads can serve on multiple teams, as much as their capacity and well-being allows.
-- CoreOps team members act as interim team leads in case team leads step down without a replacement.
-- Teams are generally comprised of volunteers. Depending on the scope and time investment of team leads, payment can be offered for ongoing service to people who have shown dedication to their role in the community.
+- Teams leads are paid positions and have a seat on the community board, which can propose and vote on internal organizational changes and processes. Team budgets are determined by the team leads and voted on by the community board.
+- CoreOps team members act as backup team leads in a case team lead steps down without a replacement.
+- Teams are generally comprised of volunteers. Depending on the scope and time investment of team members, they can step up to be a team lead after an agreed upon duration of service on the team.
 
 Joining and leaving teams
 -------------------------
 
-Team members
-  - Team members serve for a minimum of three months.
-  - Membership renews automatically after joining. To step down from the team, members need to give a 14-day notice to the team lead before the end of the term.
-  - Team membership requirements:
+  Team members
+    - Team members serve for a minimum of three months.
+    - Membership renews automatically after joining. To step down from the team, members need to give a 14-day notice to the team lead before the end of the term.
```

Comment:
While we want to avoid people joining and leaving all the time without commitment, I feel like this section isn't offering enough space for people who need to step back, temporarily or permanently, for well-being reasons, like potential burnout, needing to take care of an ill family member, losing work, etc.

Reply:
Agreed.

Reply:
Of course, we can definitely add a force majeure note.